src/css/style.css

```diff
@@ -0,0 +1,175 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  background-color: #f5f5f5;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 80vh;
+  font-size: 1.25rem;
+}
+
+.todo-container {
+  position: relative;
+  box-shadow: 0 3px 10px rgba(19, 17, 17, 0.21);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
+  margin: 0 auto;
+  width: 60%;
+}
+
+.todo-header,
+form {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem 1rem;
+  border-bottom: 2px solid #f1f1f1;
+}
+
+.todo-header {
+  padding: 1.5rem 1rem;
+}
+
+.todo-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+.todo-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem 1.5rem;
+  border-bottom: 2px solid #f1f1f1;
+}
+
+.todo-item .description {
+  width: 100%;
+  color: #6d6868;
+  font-size: 1.25rem;
+}
+
+input {
+  border: none;
+  padding: 0.8rem;
+  outline: none;
+  background-color: transparent;
+}
+
+.todoCheck {
+  display: inline-block;
+  height: 1.25rem;
+  width: 1.25rem;
+  margin-right: 16px;
+  border: 3px solid #ccc;
+  flex-shrink: 0;
+}
+
+.clear-completed {
+  background-color: #eee;
+  color: #6d6868;
+  font-size: 1.25rem;
+  border: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 2rem;
+  outline: none;
+  cursor: pointer;
+}
+
+button.hidden {
+  display: none;
+}
+
+.input-task {
+  font-style: italic;
+  font-size: 1.2rem;
+}
+
+.todo-item.list-highlight {
+  background-color: #e2e5b9;
+}
+
+button {
+  border: none;
+  background-color: transparent;
+  font-size: 1.25rem;
+  cursor: pointer;
+}
+
+.completed-task {
+  text-decoration: line-through;
+}
+
+.completed-task:hover {
+  color: #08910f;
+}
+
+.todoCheck::after {
+  content: "";
+  display: none;
+}
+
+.todoCheck.checked::after {
+  display: block;
+}
+
+.todoCheck.darken {
+  border: 2px solid #938f8f;
+}
+
+.todo-item .todoCheck.checked {
+  border: none;
+}
+
+.clear-completed:hover {
+  text-decoration: underline;
+}
+
+.todo-item .todoCheck::after {
+  position: relative;
+  top: 2px;
+  left: 7px;
+  width: 0.5rem;
+  height: 1rem;
+  border: solid #0b5d9b;
+  border-width: 0 3px 3px 0;
+  -webkit-transform: rotate(45deg);
+  -ms-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.todo-refresh,
+.todo-add {
+  color: #6d6868;
+}
```

Comment:
According to the Best practices in Css guide, I recommend that you use global variables in the root of your css or declare variable as shown in the guide below.
![image](https://github.com/mulugithub/To-Do-List-Peer-Review/assets/99537435/045afcc2-9fc8-4a6e-9772-7299b6a3583f)